static/docs/get-started/connect-code-and-data.md

```diff
@@ -113,38 +119,37 @@ This is how `prepare.dvc` looks like internally:
 ```
 
 > `dvc run` is just the first part of the DVC commands that are required to
-generate the computational graph, or in other words, an instruction how to build
-a certain ML model (data file) from another data file (or a directory). We would
-recommend to try to read a few next chapters first, before switching to other
-documents. Hopefully, `dvc run` and `dvc repro` will make more sense after
-finishing up this guide.
+generate a [pipeline's](/doc/get-started/pipeline) computational graph, or in
```

Comment:
I would mention directories here

Reply:
Sorry, I'm not understanding this comment ):

Reply:
`data file (or a directory)` from the previous version - I meant that we should probably mention whenever it makes sense that we operate on directories as well

Reply:
Got it, addressed!